src/tools/genpolicy/src/registry.rs

```diff
@@ -252,130 +256,166 @@ async fn get_image_layers(
     Ok(layers)
 }
 
-fn delete_files(decompressed_path: &Path, compressed_path: &Path, verity_path: &Path) {
-    let _ = fs::remove_file(&decompressed_path);
-    let _ = fs::remove_file(&compressed_path);
-    let _ = fs::remove_file(&verity_path);
-}
-
 async fn get_verity_hash(
     use_cached_files: bool,
     client: &mut Client,
     reference: &Reference,
     layer_digest: &str,
 ) -> Result<String> {
-    let base_dir = std::path::Path::new("layers_cache");
-
+    let temp_dir = tempfile::tempdir()?;
```

Comment:
Does this directory go under the current directory, or somewhere else?

If it goes under /tmp:

1. tmpfs might be smaller than the container image size. Bryce mentioned some huge container images in the past.
2. We might have problems similar to https://salvatoresecurity.com/the-many-perils-of-tmp/